app/views/equipments/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<h1 class='repair-header d-flex a-center j-center'>Equipment Repairs</h1>
+<%if @equipmentrepairs.empty?%>
+  <div class='equipments-show-page d-flex a-center j-center'>
+    <h1 class='index-header'>No repair for this equipment. Please add a repair <%= link_to "here", new_repair_path%></h2>
+  </div>
+<%else%>
+    <div class='repair-cont d-flex f-col a-center j-center'>
+    <%= render partial: "layouts/flash" %>
+    <div class='equipment d-flex a-center'>
+      <%total =0%>
+      <%@equipmentrepairs.each {|eqp| total +=eqp.repair.amount } %>
+      <%= image_tag("#{@equipments.icon}", width:'150px', height:'150px') %>
+            <div class='name-date d-flex f-col a-center'>
+                <p><%= @equipments.name%></p>
+                <p><%= @equipments.created_at.strftime('%d %b %Y at %I %M %p') %></p>
+            </div>
+            <h3 class='price'><%= "$#{total}"%></h3>
+    </div>
+    <div class='new-repair new-equipment d-flex a-center j-center'>
+        <%= link_to "Add new Repair", new_repair_path %>
+    </div>
+    <div class='equip-repair-cont d-flex a-center j-center f-col'>
+      <%@equipmentrepairs.each do |eq|%>
+        <div class='repair d-flex a-center'>
+        <div class='cost-date d-flex f-col'>
+           <h1><%= "Your #{eq.repair.name } repair on #{eq.equipment.name} costs: $#{eq.repair.amount} "%></h1>
+            <p><%= eq.repair.created_at.strftime('%d %b %Y at %I %M %p') %></p>
+        </div>
+           
+            <%if can? :destroy, eq.repair %>
+              <%= button_to "Delete", repair_path(eq.repair.id), :method => :delete %>
+            <%end%>
+        </div>
+    <%end%>
+    </div>
+    
+   </div>
+<%end%>
```

Comment:
- [ ] When I click on the add a repair, I am getting this error that is coming from the `repairs_controller.rb:6`, please fix it.
 
![image](https://user-images.githubusercontent.com/55635977/158817356-68865fff-0449-4e85-b45d-b5356e9aea08.png)